app/templates/instruction-forms/first-degree-assault-form.njk

```diff
@@ -1,2 +1,15 @@
 <input type="hidden" name="first_degree_assult" value="true" />
 <h1>FIRST DEGREE ASSAULT</h1>
+<div class="form-group">
+	<label> Deceased First name: <label>
+	<input type="text" name="dfname">
+</div>
+<div class="form-group">
+	<label> Deceased Last name: <label>
+	<input type="text" name="dlname">
+</div>
+<div class="form-group">
+    <label>Assault executed by?</label><br>
+	<input type="checkbox" name="Firearm" value="drugs"> Use of Firearm<br>
+	<input type="checkbox" name="Injury" value="alcohol"> Causing physical injury
```

Comment:
For the `name` attribute I think we should stick to camelcase or dash-separated with all lowercase.